scripts/tfcloud/tfcloud.sh

```diff
@@ -536,9 +536,9 @@ function make_curl_request() {
     esac
   done
 
-  command="curl -sS -L -w '%{http_code}' --header 'Authorization: Bearer xxxxxx' --header 'Content-Type: application/vnd.api+json' $options -- '"${url}"' 2> >(tee /dev/stderr)"
+  command="curl -sS -L -w '%{http_code}' --header 'Authorization: Bearer xxxxxx' --header 'Content-Type: application/vnd.api+json' $options -- '"${url}"' 2> >(tee -a >&2)"
```

Comment:
> Can you explain why you are removing the redirection? Can you also show a test evidence of the confirm the code is working?

Hi @LaurentLesle, the /dev/stderr only can write by root user. So that, the Rover Agent cannot redirect standar error 2> to /dev/stderr. 
However, **/dev/stderr** is equivalent to **/dev/fd/2**, **/proc/self/fd/2** and **&2**. In Linux, write to &2 works for every user.
I have update it to redirect to &2, the azdo pipeline pass after the fix
<img width="1043" alt="Screenshot 2023-12-19 at 11 08 58" src="https://github.com/aztfmod/rover/assets/50902568/fe0e082b-21d3-4f23-b09e-1fe1f3dd7961">

Reply:
Thanks @hanganhhung123  for taking the time to explain. Your explanation make sense.